src/index.ts

```diff
@@ -511,6 +513,29 @@ namespace Private {
         '.jp-SidePanel .jp-DirListing-content .jp-DirListing-item[data-isDir]',
       rank: 10
     });
+
+    app.commands.addCommand(CommandIDs.rename, {
+      execute: args => {
```

Comment:
I guess we don't use `args` and we rely on the widget to do all the work? Sort of funny to have a rename command that doesn't actually do the renaming.